doxia-site-model/src/main/mdo/site.mdo

```diff
@@ -66,6 +66,15 @@ under the License.
            <type>String</type>
            <defaultValue>merge</defaultValue>
         </field>
+        <field xml.attribute="true">
+           <description><![CDATA[
+             Whether to enforce a parent site descriptor.
```

Comment:
Should probably outline that this is only effective in case the the POM has a parent defined.